components/actions/ActionCardList.js

```diff
@@ -118,6 +118,12 @@ function ActionCardList({
     theme,
     t
   );
+  const actionsById = useMemo(
+    () => new Map(actions.map((a) => [a.id, a])),
+    [actions]
```

Comment:
It looks like `actions` is a new array on each render, so this memoization isn't necessary. If we notice performance issues we should refactor `ActionList`, it looks pretty messy anyway 👀